core/test/audits/third-party-summary-test.js

```diff
@@ -17,7 +17,7 @@ describe('Third party summary', () => {
     const artifacts = {
       devtoolsLogs: {defaultPass: pwaDevtoolsLog},
       traces: {defaultPass: pwaTrace},
-      URL: {finalDisplayedUrl: 'https://pwa-rocks.com'},
+      URL: {finalDisplayedUrl: 'https://pwa.rocks'},
```

Comment:
what happened here?

Reply:
That seemed to be an incorrect entry, per `progressive-app-m60.json` trace which has no hostname matching `pwa-rocks.com`.